package.json

```diff
@@ -24,6 +24,7 @@
     "gray-matter": "^4.0.3",
     "htmr": "^1.0.2",
     "i18next": "^23.6.0",
+    "lodash": "^4.17.21",
```

Comment:
Noticed that we are using the `shuffle` module. Could we try to install just that module instead of the whole lodash lib? similar as with `lodash.merge`. Could we just install `lodash.suffle` or do you think it is better to do it this way?

Reply:
Good idea, will change this